deploy/scripts/src/btm/test.ts

```diff
@@ -102,21 +254,143 @@ const awsConfig = new AWSConfig({
 })
 
 const eventData = {
-  payload: __ENV.DATA_BTM_SQS_PAYLOAD
+  payload: __ENV.DATA_BTM_SQS_PAYLOAD,
+  payloadData1: __ENV.DATA_BTM_SQS_PAYLOAD_DATA1,
+  payloadData2: __ENV.DATA_BTM_SQS_PAYLOAD_DATA2,
+  payloadData3: __ENV.DATA_BTM_SQS_PAYLOAD_DATA3,
+  payloadTimestamp: __ENV.DATA_BTM_SQS_PAYLOAD_TIMESTAMP
 }
 
+const payloadData1Array = eventData.payloadData1.split(',')
+const payloadData2Array = eventData.payloadData2.split(',')
+const payloadData3Array = eventData.payloadData3.split(',')
+const payloadTimestampArray = eventData.payloadTimestamp.split(',')
+const payloadTimestampMin: number = Number(payloadTimestampArray[0])
+const payloadTimestampMax: number = Number(payloadTimestampArray[1])
+
+let messageBody: string
+let randomTimestamp: number
+let timestampFormatted: string
+
 const sqs = new SQSClient(awsConfig)
 
-export function sendEvent (): void {
-  const messageBody = eventData.payload.replace('UUID', uuidv4())
+export function sendEvent1 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[0])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[0])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[0])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent1 === debug === messageBody \n', messageBody)
+  sqs.sendMessage(env.sqs_queue, messageBody)
+}
+
+export function sendEvent2 (): void {
+  randomTimestamp = Math.floor(Math.random() * (payloadTimestampMax - payloadTimestampMin + 1)) + payloadTimestampMin
+  timestampFormatted = new Date(randomTimestamp * 1000).toISOString()
+  messageBody = eventData.payload.replace('UUID', uuidv4())
+  messageBody = messageBody.replace('DATA1', payloadData1Array[1])
+  messageBody = messageBody.replace('DATA2', payloadData2Array[1])
+  messageBody = messageBody.replace('DATA3', payloadData3Array[1])
+  messageBody = messageBody.replace(/""timestamp"":\s*""\d+""/, `""timestamp"": ${randomTimestamp}`)
+  messageBody = messageBody.replace('TIMESTAMP', randomTimestamp.toString())
+  messageBody = messageBody.replace('TIMESTAMP_FORMATTED', timestampFormatted).replace('Z', '')
+  console.log('sendEvent2 === debug === messageBody \n', messageBody)
+  sqs.sendMessage(env.sqs_queue, messageBody)
```

Comment:
All `sendEvent#` functions are the same with the exception of the test data, all this duplication can be reduced if the data is paramaterised

Reply:
Done